test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/EndToEndTestsBase.cs

```diff
@@ -192,7 +192,8 @@ public async Task QueueTriggerToBlobTest()
             string result = await TestHelpers.WaitForBlobAndGetStringAsync(resultBlob);
             Assert.Equal(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent), TestHelpers.RemoveByteOrderMarkAndWhitespace(result));
 
-            LogMessage traceEvent = await WaitForTraceAsync(p => p?.FormattedMessage != null && p.FormattedMessage.Contains(id));
+            string userCategory = LogCategories.CreateFunctionUserCategory("QueueTriggerToBlob");
```

Comment:
FYI @brettsam , after removing the filter check, I had to add this in order to filter out the worker logs in tests, I believe this is expected, but wanted to bring it up.